litellm/tests/test_keywordsai_integration.py

```diff
@@ -0,0 +1,114 @@
+import os
+import litellm
+
+# METHOD 1: Using keywordsai as the LLM proxy
+litellm.api_base = "https://api.keywordsai.co/api/"
+KEYWORDS_AI_API_KEY = os.getenv("KEYWORDS_AI_API_KEY")
+
+response = litellm.completion(
+    api_key=KEYWORDS_AI_API_KEY, # !!!!!!! Use the keyowrdsai api key in your completion call !!!!!!!
+    model="gpt-3.5-turbo",
+    messages=[{"role": "user", "content": "How does a court case get to the Supreme Court?"}]
+)
+
+print(response)
+# Go to https://platform.keywordsai.co/ to see the log
+
+# ================================================================================================================
+# METHOD 2: Use keywordsai callback:
+
+
+
+import litellm
+from litellm import completion, acompletion
+from litellm.integrations.keywordsai import KeywordsAILogger
+# litellm.set_verbose = True
+litellm.api_base = None
+litellm.callbacks = [KeywordsAILogger()]
```

Comment:
this is now how logging integrations on litellm work. 

A user should just be able to do `litellm.callbacks = ["keywords_ai"]` and it should just work. 

https://github.com/BerriAI/litellm/blob/a3d4bf6c27cab69cb9ada11f3030657cdc16c5a4/litellm/litellm_core_utils/litellm_logging.py#L2278

Reply:
Resolved.

Sorry for the confusion, I meant to submit the collection of stream chunks. I used the existing complete_stream_response attribute from kwargs in my new pr

Reply:
Fixed